performance-test/src/gatling/java/org/opensearch/dataprepper/test/performance/TargetRpsSimulation.java

```diff
@@ -46,8 +46,6 @@ private static PopulationBuilder runScenarioWithTargetRps(final ScenarioBuilder
     {
         setUp(
                 runScenarioWithTargetRps(smallBatchScenario, 400),
-                runScenarioWithTargetRps(smallBatchScenario, 500),
```

Comment:
Are these tests no longer needed?  beyond supported perf?

Reply:
I noticed a bug. Depending on what gradle args you provide (parallel or re-run), the test may not run with a duplicate scenario name exception.